src/components/issue-description.component.js

```diff
@@ -12,62 +13,67 @@ import {
   Button,
 } from 'components';
 import { translate } from 'utils';
-import { colors, fonts, normalize } from 'config';
+import { colors, styledFonts, normalize } from 'config';
 import { v3 } from 'api';
 
-const styles = StyleSheet.create({
-  headerContainer: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'center',
-    paddingRight: 10,
-  },
-  borderBottom: {
-    borderBottomWidth: 1,
-    borderBottomColor: colors.greyLight,
-  },
-  title: {
-    color: colors.primaryDark,
-    ...fonts.fontPrimarySemiBold,
-  },
-  titleSmall: {
-    color: colors.primaryDark,
-    ...fonts.fontPrimarySemiBold,
-    fontSize: normalize(10),
-  },
-  listItemContainer: {
-    borderBottomWidth: 0,
-    flex: 1,
-  },
-  diffBlocksContainer: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'flex-end',
-    paddingRight: 10,
-    paddingBottom: 10,
-  },
-  badge: {
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-  labelButtonGroup: {
-    flexWrap: 'wrap',
-    flexDirection: 'row',
-    marginLeft: 54,
-    paddingBottom: 15,
-  },
-  assigneesSection: {
-    marginLeft: 54,
-    paddingBottom: 5,
-  },
-  mergeButtonContainer: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
-    paddingVertical: 15,
-  },
-});
+const HeaderContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding-right: 10;
+`;
+
+const ContainerBorderBottom = styled.View`
+  border-bottom-width: 1;
+  border-bottom-color: ${colors.greyLight};
+`;
+
+const ListItemStyled = styled(ListItem)`
+  color: ${colors.primaryDark}
+  font-family: ${styledFonts.fontPrimarySemiBold}
+`;
+
+const ListItemURL = ListItemStyled.extend`
+  font-size: ${normalize(10)};
+`;
+
+const ListItemIssueTitle = ListItemStyled.extend`
+  border-bottom-width: 0;
+  flex: 1;
+`;
+
+const DiffBlocksContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  padding-right: 10;
+  padding-bottom: 10;
+`;
+
+const StateBadgeStyled = styled(StateBadge)`
```

Comment:
🤔  This is failing the test for some reason.

```
Cannot create styled-component for component: undefined
```

Reply:
Actually, why are we styling the StateBadge at all? Seems a bit weird. Shouldn't this all be done in the component?

Reply:
Couldn't agree more @andrewda . I reverted these styles to use StyleSheet while we refactor StateBadge component, so it doesn't require any style props.